ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -961,9 +975,11 @@ _ts_get_gstcaps_from_conf (GstTensorSrcTIZENSENSOR * self)
     const gchar *typestr = gst_tensor_get_type_string (spec->type);
     gchar *dimstr = gst_tensor_get_dimension_string (spec->dimension);
     g_assert (typestr && typestr[0]);
-    tensor = g_strdup_printf ("other/tensor, dimension=%s, type=%s ; "
-        "other/tensors, num_tensors=1, dimensions=%s, type=%s",
-        dimstr, typestr, dimstr, typestr);
+    tensor = g_strdup_printf ("other/tensor,framerate=(fraction)%d/%d,dimension=%s,type=%s ; "
```

Comment:
For future work, let's consider gst_tensors_caps_from_config() and gst_tensor_caps_from_config() to get the caps.

Reply:
I updated the part. Please review.